lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
```

Comment:
Does `latest` means the `master` branch on GitHub or the latest version on npm?
We should either target the latest version on npm, or even better IMO, hardcode versions, but not point to GitHub's master.